run_analysis/run_analysis.R

```diff
@@ -60,23 +66,52 @@ ideal_response_from_frequency <- function(frequency_data, region_to_load, f_ulco
 }
 
 #' Apply user filters to combined data
-filter_combined_data <- function(combined_data, off_grid_postcodes, cleaned, size_groupings, standards, postcodes,
-                                manufacturers, models, sites, circuits) {
-  logdebug("Apply user filters to combined data", logger=logger)
+filter_combined_data <- function(
```

Comment:
This function definition doesn't match the style guide. I prefer to have the arguments indented to be in line with the opening bracket. It looks to follow that style in most other files but not this one.

Reply:
Thanks for catching this! I've searched and fixed all three cases where function indentation was like this